fil_fungible_token/src/runtime/messaging.rs

```diff
@@ -71,8 +71,7 @@ impl Messaging for FvmMessenger {
         params: &RawBytes,
         value: &TokenAmount,
     ) -> Result<Receipt> {
-        let params = RawBytes::new(fvm_ipld_encoding::to_vec(&params)?);
-        Ok(send::send(to, method, params, value.clone())?)
+        Ok(send::send(to, method, params.clone(), value.clone())?)
```

Comment:
👍 thanks

Reply:
I want to have a further look at this later.  I think every time we call this, we're creating a `RawBytes` at the call site to serialise some params struct. Might be better to just move ownership (of the `RawBytes`) in that case to avoid the clone and save a little gas.